bindgen/codegen/helpers.rs

```diff
@@ -56,7 +56,9 @@ pub(crate) mod attributes {
         if comment.is_empty() {
             quote!()
         } else {
-            quote!(#[doc = #comment])
+            let doc_lines =
+                comment.lines().map(|line| quote! { #[doc = #line] });
+            quote!(#(#doc_lines)*)
```

Comment:
Perhaps the function should also return a Vec<TokenStream> so we can let attr: Attribute = parse_quote!(#attr) them back to individual attributes again.